src/test/ui/associated-type-projection-from-multiple-supertraits.stderr

```diff
@@ -43,4 +43,4 @@ error[E0221]: ambiguous associated type `Color` in bounds of `C`
 error: aborting due to 4 previous errors
 
 You've got a few errors: E0191, E0221
-If you want more information on an error, try using "rustc --explain E0191"
+For more information about an error, try "rustc --explain E0191"
```

Comment:
@zackmdavis What do you think about "You've got a few errors" above?

Reply:
Also, do we have an English language expert here? Is the comma before "try" necessary?

Reply:
> What do you think about "You've got a few errors" above?

I'd prefer "There were some errors" or "Some errors occurred."

Others might disagree with my style æsthetics, but to me, the second person (addressing the user as "you") and contractions feel like too much attempted folksiness in contrast to the "serious", "no-nonsense" tone I expect from compiler error messages.

> Is the comma before "try" correct?

Yes, it's [separating a dependent clause from a subsequent independent clause](http://theeditorsblog.net/2014/07/30/commas-with-subordinate-clauses-a-readers-question/).

Reply:
I agree with @zackmdavis here.

@GuillaumeGomez 
Could you make the mentioned changes and then squash into 1 commit?

We should get this right on the first try (no grammatical errors, etc), I'd really like to avoid landing one more "+1,040 −1,042" PR potentially conflicting with any other PR changing UI tests.